features/user_can_comment.feature

```diff
@@ -0,0 +1,23 @@
+Feature:
+"As a visitor,
```

Comment:
fix the indentation for the user story and add a feature title on line 1

Reply:
Fixed